ci-operator/step-registry/ofcir/acquire/ofcir-acquire-commands.sh

```diff
@@ -76,15 +76,17 @@ function getCIR(){
     IPFILE=$SHARED_DIR/server-ip
     CIRFILE=$SHARED_DIR/cir
 
-    if ! timeout 10s curl -kfX POST -H "X-OFCIRTOKEN: $OFCIRTOKEN" "$OFCIRURL?name=$JOB_NAME/$BUILD_ID&type=$CIRTYPE" -o $CIRFILE ; then
```

Comment:
Did you intend to remove timeout here? Maybe set it to 120  to at least make sure the job does not hang forever?

Reply:
Yes, I did intend to remove it, it seemed pointless if retrying after 60 seconds, but your correct, I'll put it back in with an increased timeout